p4app-exercises/basic.p4app/main.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, n, **opts):
         sw.insertTableEntry(table_name='MyIngress.ipv4_lpm',
                             match_fields={'hdr.ipv4.dstAddr': ["10.0.0.%d" % j, 32]},
                             action_name='MyIngress.ipv4_forward',
-                            action_params={'dstAddr': '00:00:00:00:00:%02x' % j,
+                            action_params={'dstAddr': '00:00:00:00:ff:%02x' % j,
```

Comment:
Any particular reason for changing the Ethernet addresses?  Maybe just to make them look more different from the IPv4 addresses?

Reply:
This rule is for switch to switch forwarding and the dstAddr here is the one of the switch
I think it's more sensical to use different mac addresses for switches & hosts since this exercise is supposed to be L3 forwarding
I realized that something weird was going on because when using `receive.py` packets were showing up with ethernet.srcAddr == ethernet.dstAddr...